### PR TITLE
$node.tags() API to iterate tags on a node

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/lib/ast.py [hunk 1]
```diff
@@ -347,52 +349,68 @@ def getRuntVars(self, runt):
...
-                    broke = True
-                    if e.item is not None:
-                        yield e.item
-                    break
+                except StormContinue:
+                    continue
+
+            yield node, path
+
+        # no nodes and a runt safe value should execut once
```
Comment (jnwatson) [inline synapse/lib/ast.py]: execut->execute

File: synapse/lib/ast.py [hunk 2]
```diff
@@ -1747,15 +1744,16 @@ class EditTagAdd(Edit):
 
     async def run(self, runt, genr):
 
-        name = await self.kids[0].runtval(runt)
+        #name = await self.kids[0].runtval(runt)
```
Comment (jnwatson) [inline synapse/lib/ast.py]: delete

File: synapse/lib/ast.py [hunk 3]
```diff
@@ -1769,11 +1767,14 @@ class EditTagDel(Edit):
 
     async def run(self, runt, genr):
 
-        name = await self.kids[0].runtval(runt)
-        parts = name.split('.')
+        #name = await self.kids[0].runtval(runt)
```
Comment (jnwatson) [inline synapse/lib/ast.py]: delete

File: synapse/lib/ast.py (hunk 1 above)
Comment (invisig0th) [reply synapse/lib/ast.py]: uh... *shifty eyes* i totally just left that to um... make sure you guys were... paying attention. :D

**Pull request merged**